Translators/WZDx/tim_generator.py

```diff
@@ -25,16 +25,35 @@ def get_duration_time_minutes(feature):
     duration = (end_date - start_date).total_seconds() / 60
     return int(duration) if duration < 32000 else 32000
 
+def calculate_anchor(p1, p2):
+    # assumes (lat, long) tuples

```

Comment:
Let's add in a method description and some comments about what we're doing. I'm not sure that that 111195 value is for instance.

Reply:
Added comments for this method that describe what is being done to calculate anchor point